test/jdk/java/net/httpclient/HttpRequestNewBuilderTest.java

```diff
@@ -121,6 +121,7 @@ record NamedAssertion(String name, BiConsumer<HttpRequest, HttpRequest> test) {
                         .headers("testName1", "z").build() },
                 // dedicated method
                 { HttpRequest.newBuilder(URI.create("https://method-1/")).GET().build() },
+                { HttpRequest.newBuilder(URI.create("https://method-1/")).HEAD().build() },
```

Comment:
I'd suggest to change the URI path to `/method-0` and move the HEAD case to just after line 122 (before GET)

Reply:
Done in the updated version of the PR